packages/data-stores/src/site/actions.ts

```diff
@@ -349,6 +345,29 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		}
 	}
 
+	function* createCustomTemplate(
+		siteSlug: string,
+		stylesheet: string,
+		slug: string,
+		title: string,
+		content: string
+	) {
+		yield wpcomRequest( {
+			path: `/sites/${ encodeURIComponent( siteSlug ) }/templates`,
+			apiNamespace: 'wp/v2',
+			body: {
+				id: `${ stylesheet }//wp-custom-template-${ slug }`,
```

Comment:
I found we don't need to send the id to API because it consists of a theme and a slug automatically.

Reply:
Wow, you're right, thanks! Removed the id field 😄